rita_common/src/rita_loop/fast_loop.rs

```diff
@@ -273,7 +271,7 @@ pub fn start_rita_fast_loop() {
 /// Manages gateway functionality and maintains the gateway parameter, this is different from the gateway
 /// identification in rita_client because this must function even if we aren't registered for an exit it's also
 /// very prone to being true when the device has a wan port but no actual wan connection.
-fn manage_gateway() {
+pub fn manage_gateway() {
```

Comment:
since this is only called in one place, maybe move it to rita_client rather than making it public?

Reply:
fixed